.github/workflows/all_solutions.yml

```diff
@@ -518,3 +519,29 @@ jobs:
             C:\IntegrationTestWorkingDirectory\TestResults\**\*TestResults.xml
           if-no-files-found: error
           
+  combine-integration-test-results:
```

Comment:
Alex, double checking... this doesn't remove the results still being in the namespace artifacts?  I've found that to be handy.

Reply:
No, they'll still be there.